invoice/invoice.proto

```diff
@@ -12,37 +12,142 @@ import "google/protobuf/timestamp.proto";
 // InvoiceData is the default invoice schema
 message InvoiceData {
   // invoice number or reference number 
-  string invoice_number = 1;
+  string number = 1;
   // invoice status
-  string invoice_status = 2;
-  // name of the sender company 
-  string sender_name = 3;
-  // street and address details of the sender company
-  string sender_street = 4;
-  string sender_city = 5;
-  string sender_zipcode = 6;
-  // country ISO code of the sender of this invoice 
-  string sender_country = 7;
-  // name of the recipient company 
-  string recipient_name = 8;
-  string recipient_street = 9;
-  string recipient_city = 10;
-  string recipient_zipcode = 11;
-  // country ISO code of the receipient of this invoice 
-  string recipient_country = 12;
-  // ISO currency code 
-  string currency = 13;
-  // invoice amount including tax 
-  bytes gross_amount = 14;
-  // invoice amount excluding tax 
-  bytes net_amount = 15;
-  bytes tax_amount = 16;
-  bytes tax_rate = 17;
-  bytes recipient = 18;
-  bytes sender = 19;
-  bytes payee = 20;
-  string comment = 21;
-  google.protobuf.Timestamp due_date = 22;
-  google.protobuf.Timestamp date_created = 23;
-  bytes extra_data = 24;
+  string status = 2;
```

Comment:
I remember that some of the fields had more comments about their use in the ticket, did we remove?